satisfactory/egg-pterodactyl-satisfactory.json

```diff
@@ -102,12 +142,22 @@
         },
         {
             "name": "[Advanced] Satisfactory Dedicated Server App ID",
-            "description": "Steam App ID used for installation and updates.",
+            "description": "Steam App ID used for installation and updates. Rarely needs to be changed.",
             "env_variable": "SRCDS_APPID",
             "default_value": "1690800",
             "user_viewable": false,
             "user_editable": false,
-            "rules": "required|integer|in:1690800",
+            "rules": "required|integer|min:1",
+            "field_type": "text"
+        },
+        {
+            "name": "Disable Seasonal Events",
```

Comment:
Part of the new Server GUI

Reply:
You can only disable it in Advanced Game Settings, which disables Steam achievements.  The way here allows you to still get achievements, but have Seasonal Events disabled.

Reply:
This brings up a good point, but I also feel like I'm going crazy... I removed this Startup Variable and updated the README to say that it was now managed by the in-game Server Manager when the game was v1.0.0, because I could have swore I saw it as an option under the "Server Settings" tab. But now, on v1.0.2, I'm taking a look, and it's not an option under this tab ***nor*** is it an option under "Create Game -> Advanced Game Settings". Did they remove this setting with the recent patches? I guess in the end it might just be better for us to wait for Fixmas to come along and see if `-DisableSeasonalEvents` still allows achievements, or if the setting will pop back up as a server option.

Reply:
Now it's back? I swear this wasn't here when I wrote my last comment...
![image](https://github.com/user-attachments/assets/f349bc00-ee87-4f04-98bb-482176b868b6)